indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -479,9 +479,10 @@ class BookingDetails extends React.Component {
     const showActionButtons = !isCancelled && !isRejected && (canCancel || canReject || showAccept);
     const activeBookings = _.omitBy(occurrences.bookings, value => _.isEmpty(value));
     const occurrenceCount = Object.keys(activeBookings).length;
+
     return (
       <>
-        <Modal onClose={onClose} size="large" closeIcon open>
+        <Modal onClose={() => onClose()} size="large" closeIcon open>
```

Comment:
why was this needed? the arguments passed by the modal causing problems?

Reply:
discussed in person